src/index.js

```diff
@@ -4,11 +4,12 @@ var Debugger = require('./ui/Ethdebugger')
 var BasicPanel = require('./ui/BasicPanel')
 var TraceManager = require('./trace/traceManager')
 var CodeManager = require('./code/codeManager')
+var SourceMappingDecoder = require('./util/sourceMappingDecoder')
 
 if (typeof (module) !== 'undefined' && typeof (module.exports) !== 'undefined') {
   module.exports = modules()
 }
-if (window) {
+if (typeof (window) !== 'undefined') {
   window.remix = modules()
```

Comment:
the thing is that `src/index.js` can be loaded using two different ways:
 - using require (bowserify handle everything right)
 - it might be that we put `src/index.js` inside a script tag . in that case there is no `module.exports`